src/main/scala/com/nawforce/commands/DependencyExplorer.scala

```diff
@@ -89,7 +89,7 @@ class DependencyExplorer(context: ExtensionContext, server: Server) {
               panel.webview.postMessage(
                 new InitMessage(isTest = false,
                                 identifier.toString(),
-                                typeIdentifiers.identifiers.map(_.toString()).toJSArray))
+                                typeIdentifiers.identifiers.map(_.toString()).filter(TypeIdentifier(_).isRight).toJSArray))
```

Comment:
This is a work around to ensure the strings used in the drop down can be used to create a type identifier.